### ISSUE TITLE
Increment/decrement tool size with keyboard shortcuts

### ISSUE THREAD

**Issue by maoschanz** [maoschanz/drawing#242]
## Use case

the widget for tool size is a quite small target for the mouse, and is almost the only thing not accessible with kb shortcuts

## Suggested solution

- <kbd>alt</kbd>+<kbd>up</kbd> to increase
- <kbd>alt</kbd>+<kbd>down</kbd> to decrease

### Possible alternatives

change the controls to scroll to <kbd>alt</kbd>+<kbd>up/down</kbd>, and use <kbd>ctrl</kbd>+<kbd>up/down</kbd> for this feature
